PR-Template.txt

```diff
@@ -0,0 +1,7 @@
+[ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.

```

Comment:
Looks like start point is https://issues.apache.org/jira/projects/CAMEL/issues

Reply:
CAMEL-13704 is the one I thought I could work on, do I have to do something to claim the issue?